torch/ao/quantization/fx/utils.py

```diff
@@ -152,6 +153,16 @@ def get_qconv_prepack_op(conv_op: Callable) -> Callable:
     assert prepack_op, "Didn't find prepack op for {}".format(conv_op)
     return prepack_op
 
+def get_qconv_transpose_prepack_op(conv_transpose_op: Callable) -> Callable:
```

Comment:
nit: it feels like we can merge this helper function with get_qconv_prepack_op and just have a single helper function?